static/js/recent_topics_ui.js

```diff
@@ -649,6 +650,7 @@ export function show() {
     narrow_state.reset_current_filter();
     narrow.set_narrow_title("Recent topics");
     message_view_header.render_title_area();
+    pm_list.handle_narrow_deactivated();
```

Comment:
This should go next to `stream_list.handle_narrow_deactivated`, above, unless there's a very good reason for it not to.

Reply:
Added a comment to answer the reason in #21726